src/Microsoft.Health.Dicom.Client/Models/AddExtendedQueryTagEntry.cs

```diff
@@ -0,0 +1,34 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Client.Models
+{
+    /// <summary>
+    /// External representation of a extended query tag entry.
+    /// </summary>
+    public class AddExtendedQueryTag
+    {
+        /// <summary>
+        /// Path of this tag. Normally it's composed of groupid and elementid.
+        /// E.g: 00100020 is path of patient id.
+        /// </summary>
+        public string Path { get; set; }
+
+        /// <summary>
+        /// VR of this tag.
+        /// </summary>
+        public string VR { get; set; }
+
+        /// <summary>
+        /// Level of this tag. Could be Study, Series or Instance.
+        /// </summary>
+        public string QueryTagLevel { get; set; }
```

Comment:
* Should we use same name on Level for both AddExtendedQueryTag and  GetExtendedQueryTag?
* Curious why QueryTagLevel instead of Level? Since if we use QueryTagLevel here, we may want to append QueryTag to all other properties -- QueryTagPath, QueryTagVR etc.
* Wondering if we could use enum QueryTagLevel instead of string, and use https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_Converters_StringEnumConverter.htm for to serialize to string instead of integer. The converter could be specified on class Property or Enum or JsonSerializerSettings, Personal preference is JsonSerializerSettings, can be setup here: https://github.com/microsoft/dicom-server/blob/243437502e472f0a7e65d4b96081f474e1c90b28/src/Microsoft.Health.Dicom.Client/DicomWebClient.cs#L43
* Some thought: are AddExtendedQueryTag and GetExtendedQueryTag good names? They are more like some actions instead of data model, but I don't have good candidates, maybe _AddExtendedQueryTagEntry_ and _ExtendedQueryTagEntry_ ? I'm not sure.

Reply:
Reverted back to just 'Level' everywhere.

I added the enum in the client but I prefer a serializer specific to the class to ensure other enums are serialized in the way they need to be.

I think these names work for now...I'll keep it in mind and see if longer term I have a better name for the two entries!